resolwe_api/resolwe.py

```diff
@@ -189,16 +213,18 @@ def processes(self, process_name=None):
         :rtype: list of Process objects
 
         """
+        # TODO: Make a better representation? If this is run from command line
+        # it outputs thousands of lines... Not really helpful.
         if process_name:
             return [p for p in self.api.process.get() if process_name in p['name']]
         else:
             return self.api.process.get()
 
-    def printad_processes(self):
+    def print_upload_processes(self):
         """Print all upload process names."""
         for p in self.processes():
             if 'upload' in p['category']:
-                print(p['name'])
+                sys.stdout.write(p['name'] + '\n')
```

Comment:
I thought that it is not possible to mock print function. When I was running tests, I ended up with a bunch of print statements in the terminal... Only now I see that it is possible to mock print. Should I change all stdout to print?